tmt-app/src/Components/QuizButtonContainer.jsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Link, Typography } from '@mui/material';
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+
+import { PrimaryBtn } from '../Styling/CustomStyling.js';
+function QuizButtonContainer({ step }) {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        alignContent: 'center',
+        justifyContent: 'space-evenly',
+        margin: '4.5rem 2rem',
+      }}
+    >
+      <Link
+        sx={{ color: 'Boba.main' }}
+        underline="none"
+        aria-label="quiz-backward"
+        component={RouterLink}
+        to="/quiz/skills"
+      >
+        <PrimaryBtn
+          sx={{
+            backgroundColor: 'white',
+            color: 'Matcha.main',
+            border: '3px solid #73956F',
+            padding: {
+              xs: '1rem 1.75rem',
+              md: '1.25rem 4.25rem',
+            },
+            '&:hover': {
+              color: 'white',
+              border: '3px solid #732C2C',
+            },
+          }}
+        >
+          <Typography variant="h4" sx={{ visibility: 'hidden' }}>
+            {step === 'classes' ? 'Next Question' : "I'm Finished"}
+          </Typography>
+          <Typography variant="h4" sx={{ position: 'absolute' }}>
+            Back
+          </Typography>
```

Comment:
What was your reason for choosing two Typography elements like this?

Reply:
![image](https://github.com/BobaTalks/pick-a-major/assets/12107969/ec606d11-65f8-4cf3-82e2-0ec614b78251)

Populating the dom with multiple elements like this can create accessibility issues and is discouraged. Read up on the nuances of `visibility: hidden` as well too. Using react, we have access to much more robust logic that can help produce a very clean DOM and not have to play with CSS

Reply:
This is my bad, I forgot about accessibility. This was to create equal size buttons without having to set width.